packages/components/src/components/RunHeader/RunHeader.jsx

```diff
@@ -12,29 +12,30 @@ limitations under the License.
 */
 
 import { useIntl } from 'react-intl';
-import { CopyButton, SkeletonPlaceholder } from '@carbon/react';
-import { copyToClipboard } from '@tektoncd/dashboard-utils';
-
-import FormattedDate from '../FormattedDate';
+import { SkeletonPlaceholder } from '@carbon/react';
+import RunTimeMetadata from '../RunTimeMetadata';
```

Comment:
Code for this component appears to be missing

Reply:
`RunTimeMetadata` component was merged previously as part of the first set of changes